sandboxes/all.yaml

```diff
@@ -9,6 +9,9 @@ spec:
     team: "@{team}"                           # key, value
   cluster: staging                            # cluster in which this sandbox is to reside.
   description: "@{dev}'s work on feature-x"   # a short description
+  ttl:                                        # optional sandbox lifetime limit
+    duration: 2d                              # N<unit>, unit m: minutes, h: hours, d: days, w: weeks
+    offsetFrom: createdAt                     # what the duration is relative to (createdAt).
```

Comment:
Is `createdAt` the only valid `offsetFrom` right now?

Reply:
Yes, that is correct